airflow/dags/resources/stages/parse/table_definitions/chainlink/view_AccessControlledOffchainAggregator_info.sql

```diff
@@ -110,4 +110,9 @@ STRUCT('0xf7c72effe3975c25252539685574a5f5bde19423' as contract_address, 4 as co
 STRUCT('0xf9c53a834f60cbbe40e27702276fbc0819b3afad' as contract_address, 4 as contract_version, 9 as decimal_places, 'USDC / USD' as name, ('USDC','USD') as pair, '0xfe4a8cc5b5b2366c1b58bea3858e81843581b2f7' as proxy_address, 'Circle USD' as asset_name, 'verified' as feed_category, 'Crypto' as feed_type, 8 as decimals, False as single_source, '$' as symbol, '2020-10-27' as date_added),
 STRUCT('0xfc8fb8bd285ff358fef481b45dbc7450c0f8f89d' as contract_address, 4 as contract_version, NULL as decimal_places, 'ANT / USD' as name, ('','') as pair, '0x213b030e24c906ee3b98ec7538cc6d3d3c82af55' as proxy_address, 'Aragon' as asset_name, 'verified' as feed_category, 'Crypto' as feed_type, 8 as decimals, False as single_source, '$' as symbol, '2022-09-08' as date_added),
 STRUCT('0xfd001350eac4bed88dde16d0770d12e8e4d9508b' as contract_address, 4 as contract_version, NULL as decimal_places, 'GNO / USD' as name, ('','') as pair, '0x432fa0899cf1bcdb98592d7eaa23c372b8b8ddf2' as proxy_address, 'Gnosis' as asset_name, 'verified' as feed_category, 'Crypto' as feed_type, 8 as decimals, False as single_source, '$' as symbol, '' as date_added),
-STRUCT('0xfd54f97a6c408561b5df798c04ae08b27ca0d7f7' as contract_address, 4 as contract_version, NULL as decimal_places, 'KRW / USD' as name, ('','') as pair, '0x24b820870f726da9b0d83b0b28a93885061dbf50' as proxy_address, 'Korean Won' as asset_name, 'verified' as feed_category, 'Forex' as feed_type, 8 as decimals, False as single_source, '$' as symbol, '' as date_added)])
\ No newline at end of file
+STRUCT('0xfd54f97a6c408561b5df798c04ae08b27ca0d7f7' as contract_address, 4 as contract_version, NULL as decimal_places, 'KRW / USD' as name, ('','') as pair, '0x24b820870f726da9b0d83b0b28a93885061dbf50' as proxy_address, 'Korean Won' as asset_name, 'verified' as feed_category, 'Forex' as feed_type, 8 as decimals, False as single_source, '$' as symbol, '' as date_added),
+STRUCT('0x89c991cbc41af1a0294f79947ad71a028bf164b7' as contract_address, 4 as contract_version, 0 as decimal_places, 'CRV / USD' as name, 'CRV / USD' as pair, '0x336584C8E6Dc19637A5b36206B1c79923111b405' as proxy_address, 'CRV (PoS)' as asset_name, '' as feed_category, 'Crypto' as feed_type, '' as single_source, 'CRV' as symbol, '' as date_added, 8 as decimals, 'CRV / USD' as onchain_description),
```

Comment:
```suggestion
STRUCT('0x89c991cbc41af1a0294f79947ad71a028bf164b7' as contract_address, 4 as contract_version, 0 as decimal_places, 'CRV / USD' as name, ('CRV', 'USD') as pair, '0x336584C8E6Dc19637A5b36206B1c79923111b405' as proxy_address, 'CRV (PoS)' as asset_name, '' as feed_category, 'Crypto' as feed_type, '' as single_source, 'CRV' as symbol, '' as date_added, 8 as decimals, 'CRV / USD' as onchain_description),
```

Pair is of type `struct<string, string>`

Reply:
Fixed

Reply:
I am still seeing `'CRV / USD' as pair`